### DISCUSSION TITLE
How to run the ifm-O3X120 camera ros driver in a docker container

### DISCUSSION THREAD

**Discussion by Lyk77** [ifm/ifm3d#400] [Q&A]
Our AGV industrial computer uses Ubuntu18.04 system, on which ros image is used to create docker container. We need to put the ros driver of ifm-O3X120 camera into the container to run, may I ask how to execute it

**Comment by lola-masson** [comment]
See https://github.com/ifm/ifm3d/issues/399#issuecomment-1598528083
